postinstall.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# this shell script will need wsl on windows, but bina already requires that
+
+bina tdewolff/minify --install-dir node_modules/.bin
+
+# TODO this isn't ideal but i can't find any other way to do it
+cargo install minifier --no-track --root node_modules/.bin/cargo
+echo "note: ignore the cargo warnings about PATH, that's corrected automatically"
+
+# move cargo binaries
+mv node_modules/.bin/cargo/bin/* node_modules/.bin
+rm -r node_modules/.bin/cargo
```

Comment:
I did this with tdewolff/minify but I think it's fine not to. Just reference the explicit binary path from the JS.

Reply:
makes sense but if it's not going there i feel like it should be in a more sensible directory, and i'm not about to decide where that should be

Reply:
What's the default?

Reply:
the default would be to install on the user's system in a location that depends on the operating system. if someone didn't know it had been installed and just deleted the repository it would still be there.

some directory in the repo needs to be chosen, same with the temporary input/output files i mentioned before and still need an answer for